hatchet.lock

```diff
@@ -1 +1,3 @@
---- []
+---
+- - test/fixtures/repos/github/php-getting-started
+  - master
```

Comment:
We should be able to switch this to `main` as long as heroku_hatchet is updated to 7.1.2 or later. Last release is 7.1.4